### PR TITLE
CDAP-4908 remove table property from db source

### PR DESCRIPTION
**Pull request by albertshau**
The table name is not required at all. It makes especially little
sense when the import query contains multiple tables.

### CODE REVIEW

File: database-plugins/src/main/java/co/cask/hydrator/plugin/db/batch/sink/DBSink.java [hunk 1]
```diff
@@ -191,6 +192,9 @@ private void setResultSetMetadata() throws Exception {
   public static class DBSinkConfig extends DBConfig {
     @Description("Comma-separated list of columns in the specified table to export to.")
     public String columns;
+
+    @Description("Name of the database table to use as a source or a sink.")
```
Comment (gsps1) [inline database-plugins/src/main/java/co/cask/hydrator/plugin/db/batch/sink/DBSink.java]: just sink right?

File: database-plugins/src/main/java/co/cask/hydrator/plugin/db/batch/sink/DBSink.java [hunk 2]
```diff
@@ -95,7 +95,8 @@ public void prepareRun(BatchSinkContext context) {
     // make sure that the table exists
     try {
       Preconditions.checkArgument(
-        dbManager.tableExists(driverClass), "Table %s does not exist. Please check that the 'tableName' property " +
+        dbManager.tableExists(driverClass, dbSinkConfig.tableName),
+        "Table %s does not exist. Please check that the 'tableName' property " +
```
Comment (gsps1) [inline database-plugins/src/main/java/co/cask/hydrator/plugin/db/batch/sink/DBSink.java:99]: but tableName property is removed right? 

Edit : added for sink, alright.

File: database-plugins/src/main/java/co/cask/hydrator/plugin/db/batch/sink/DBSink.java (hunk 1 above)
Comment (albertshau) [inline database-plugins/src/main/java/co/cask/hydrator/plugin/db/batch/sink/DBSink.java]: good catch, will change.

**Comment by gsps1** [comment]
lgtm

**Pull request merged**